pkg/get/get_test.go

```diff
@@ -28,6 +30,50 @@ func getTool(name string, tools []Tool) *Tool {
 	return tool
 }
 
+func Test_MakeSureToolsAreSorted(t *testing.T) {
+	got := Tools{
+		{
+			Owner: "roboll",
+			Repo:  "helmfile",
+			Name:  "helmfile",
+		},
+		{
+			Owner: "kubernetes",
+			Repo:  "kubernetes",
+			Name:  "kubectl",
+		},
+		{
+			Owner: "digitalocean",
+			Repo:  "doctl",
+			Name:  "doctl",
+		},
+	}
+
+	sort.Sort(got)
+
+	want := Tools{
+		{
+			Owner: "digitalocean",
+			Repo:  "doctl",
+			Name:  "doctl",
+		},
+		{
+			Owner: "roboll",
+			Repo:  "helmfile",
+			Name:  "helmfile",
+		},
+		{
+			Owner: "kubernetes",
+			Repo:  "kubernetes",
+			Name:  "kubectl",
+		},
+	}
+
+	if !reflect.DeepEqual(got, want) {
```

Comment:
For the size of the test, I would prefer explicit comparisons of each index, but will take this.